src/js/components/dataDives/shared/MainCards.jsx

```diff
@@ -55,7 +54,7 @@ const MainCards = ({ contentObject }) => {
                                 imageColor={card.color}
                                 heading={<h2>{card.heading}</h2>}
                                 text={cardContent}
-                                button={<ExternalLink url={card.link} onClick={() => analyticsEvent(<h2>{card.text}</h2>)}>See Project&nbsp;&nbsp;</ExternalLink>} />
```

Comment:
these are not government links

Reply:
fixed. my bad I assumed it was all external links.